full-moon/src/ast/parsers.rs

```diff
@@ -2260,6 +2260,7 @@ make_op_parser!(CompoundOp, ParseCompoundOp,
         MinusEqual,
         StarEqual,
         SlashEqual,
+        DoubleSlashEqual,
```

Comment:
Standard is `TwoSlashEqual`

Reply:
It’d be ideal if it’s consistent with its counterpart `DoubleSlash`. Should that be changed to `TwoSlash` then?

Reply:
Ah i made a mistake when naming double slash originally. That's annoying...

Yeah I think it's better to stay related to DoubleSlash. Renaming to TwoSlash is a breaking change

Reply:
Fair enough then don't change anything but we should deprecate both at some point